### PR TITLE
add burst metadata: burst_duration & swath_name

### PR DESCRIPTION
**Pull request by yunjunz**
This PR adds two more metadata to the burst class.

+ s1_slc_burst: add two more metadata:
   - burst_duration
   - swath_name

+ README: update usage to load_bursts()

### CODE REVIEW

File: src/s1reader/s1_burst_slc.py [hunk 1]
```diff
@@ -447,10 +447,27 @@ def sensing_stop(self):
...
+    @property
+    def burst_duration(self):
+        '''Returns burst sensing duration as float in seconds.
+
+        Returns:
+        --------
+        _ : float
+            Burst sensing duration as float in seconds.
+        '''
+        return self.azimuth_time_interval * self.shape[0]
```
Comment (LiangJYu) [inline src/s1reader/s1_burst_slc.py:459]:
```suggestion
        return self.azimuth_time_interval * self.length
```
Not 100% on the following: I think a name is clearer than indexing a tuple.

**Review by LiangJYu — APPROVED** [review]
LGTM

**Review by hfattahi — APPROVED** [review]
LGTM!

**Review by vbrancat — APPROVED** [review]
LGTM.

**Pull request merged**